cuebot/Dockerfile-spi

```diff
@@ -0,0 +1,79 @@
+# -----------------
```

Comment:
Please remove our spi specific Dockerfile from this PR

Reply:
Removed